CHANGELOG.md

```diff
@@ -21,11 +21,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 
 - Setting attributes with a `compute_` method will now raise an `AttributeError` https://github.com/Textualize/textual/issues/2383
+- Unknown psuedo-selectors will now raise a tokenizer error (previously they were silently ignored) https://github.com/Textualize/textual/pull/2445
+- Breaking change: `DirectoryTree.FileSelected.path` is now always a `Path` https://github.com/Textualize/textual/issues/2448
+- Breaking change: `Directorytree.load_directory` renamed to `Directorytree._load_directory` https://github.com/Textualize/textual/issues/2448
 - Unknown pseudo-selectors will now raise a tokenizer error (previously they were silently ignored) https://github.com/Textualize/textual/pull/2445
 
 ### Added
 
 - Watch methods can now optionally be private https://github.com/Textualize/textual/issues/2382
+- Added `DirectoryTree.path` reactive attribute https://github.com/Textualize/textual/issues/2448
+- Added `DirectoryTree.FileSelected.node`
+- Added `DirectoryTree.reload` https://github.com/Textualize/textual/issues/2448
```

Comment:
I usually group additions like this in a single list item to save some vertical space.
I've noticed you usually put one addition per list item. Why? I'm considering changing the way I list things in the changelog.

Reply:
For me, if all of the above were in one long single item, I'd be more prone to not pick up on the impact of the changes. Given the above changes aren't really related (so, it's not like "added `foo` support to `Bar`, `Baz` and `Wibble`) I feel it makes more sense and highlights they're different things with different impacts.

Reply:
I get your point. Thanks.